io/input/CMakeLists.txt

```diff
@@ -29,5 +29,12 @@ if(MONGOCLIENT_FOUND)
     set(input_cpps ${input_cpps} mongodb_split.cpp mongodb_inputformat.cpp)
 endif(MONGOCLIENT_FOUND)
 
+if(THRIFT_FOUND)
+    add_subdirectory(flume_connector)
+    include_directories(${THRIFT_INCLUDE_DIR})
+    add_definitions(-DWITH_THRIFT)
+    set(input_cpps ${input_cpps} flume_connector/flume_types.cpp flume_connector/flume_constants.cpp flume_connector/ThriftSourceProtocol.cpp flume_inputformat.cpp)
```

Comment:
There's a CMakeLists.txt in flume_connector dir. So some cpps can be setup there?